### PR TITLE
[e2e-tests]: Don't run all storage-req tests in sig-storage jobs

### PR DESCRIPTION
**Pull request by akalenyu**
Signed-off-by: Alex Kalenyuk <akalenyu@redhat.com>

**What this PR does / why we need it**:
Basically following up on 2 commits:
https://github.com/kubevirt/kubevirt/pull/7194/commits/5a630968c4a6dcff841074d92688a2cff8015813 https://github.com/kubevirt/kubevirt/pull/8242/commits/fa52935ce06a4c1f8d9daa12e6f4c1f049c7a924

I believe we did not intend for sig-compute tests to be run on sig-storage and vice-versa (this direction was handled in 8242), so let's stop doing that. storage-req tests that are under other sigs will continue running.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: automation/test.sh [hunk 1]
```diff
@@ -385,7 +385,7 @@ if [[ -z ${KUBEVIRT_E2E_FOCUS} && -z ${KUBEVIRT_E2E_SKIP} ]]; then
   elif [[ $TARGET =~ sig-network ]]; then
     export KUBEVIRT_E2E_FOCUS="\\[sig-network\\]"
   elif [[ $TARGET =~ sig-storage ]]; then
-    export KUBEVIRT_E2E_FOCUS="\\[sig-storage\\]|\\[storage-req\\]"
```
Comment (iholder-redhat) [inline automation/test.sh:388]: I see that now we never export `KUBEVIRT_E2E_FOCUS` with `storage-req`, so no lane will run these tests?
Am I missing anything?

Comment (akalenyu) [reply automation/test.sh:388]: They are subsets of other sigs, I tried to document this tag's purpose here in a previous PR
https://github.com/kubevirt/kubevirt/blob/4f637f49be97e6a5db5f0309930caf99597c774e/tests/README.md?plain=1#L26-L28

**Comment by xpivarc** [comment]
I think it's better to wait for labels so we can tell ginkgo:
1. I want all test that requires rwo fs to skip `!rwofilesystem-required`
2. I am ok if you skip upload tests that require the above `!(upload&&rwofilesystem-required)`

**Comment by akalenyu** [comment]
@fossedihelm WDYT?

**Comment by fossedihelm** [comment]
/lgtm
As @xpivarc we can also rename the label, maybe in a followup PR
Thank you

**Review by xpivarc — APPROVED** [review]
/approve

**Pull request merged**